app/code/Magento/Reports/Block/Adminhtml/Sales/Bestsellers/Grid.php

```diff
@@ -40,20 +41,30 @@ public function getResourceCollectionName()
     }
 
     /**
-     * {@inheritdoc}
+     * @inheritdoc
+     */
+    protected function _addCustomFilter($collection, $filterData)
+    {
+        $collection->setRatingLimit($filterData->getRatingLimit());
+
+        return $this;
+    }
+
+    /**
+     * @inheritdoc
      */
     protected function _prepareColumns()
     {
         $this->addColumn(
             'period',
             [
-                'header' => __('Interval'),
-                'index' => 'period',
-                'sortable' => false,
-                'period_type' => $this->getPeriodType(),
-                'renderer' => \Magento\Reports\Block\Adminhtml\Sales\Grid\Column\Renderer\Date::class,
-                'totals_label' => __('Total'),
-                'html_decorators' => ['nobr'],
+                'header'           => __('Interval'),
```

Comment:
Please revert formatting changes. We don`t need to align array key => values

Reply:
Done. Associative arrays alignment was removed.